common/config/rush/browser-approved-packages.json

```diff
@@ -98,6 +98,14 @@
       "name": "scheduler",
       "allowedCategories": [ "vscode-extensions" ]
     },
+    {
+      "name": "stream-chain",
+      "allowedCategories": [ "libraries", "tests", "vscode-extensions" ]
+    },
+    {
+      "name": "stream-json",
+      "allowedCategories": [ "libraries", "tests", "vscode-extensions" ]
+    },
```

Comment:
These should go in `nonbrowser-approved-packages.json`

Reply:
Done in [e8b979c](https://github.com/microsoft/rushstack/pull/4668/commits/e8b979ccdd5f37686ce3b576c35bdc0511c7544d)